web/src/engine/transformers/BookmarkConverter.ts

```diff
@@ -37,6 +37,7 @@ export const legacyWebsiteIdentifierMap = new Map([
     [ 'manhuaes', 'manhuaaz' ],
     [ 'manhuascan', 'kaliscan' ],
     [ 'manhwaclub', 'manhwahentai' ],
+    [ 'neteasecomic', 'bilibilimanhua' ],
```

Comment:
Are there any other bilibili sites that a distinction for manhua is necessary?

Reply:
* Force of the habit.
* English site closed, but a fan website is opened and supported in Haruneko. It has two names but people may refer to it as "Bilibili" or "Bilibilicomics" or "new unrelated name i forgot".  Better avoiding confusion.